web-dev/src/components/DetailScreen/comment-item.js

```diff
@@ -1,9 +1,11 @@
 import {useState} from "react";
 import {Link} from "react-router-dom";
+import {useAuth} from "../../contexts/auth-context";
 
 const CommentItem = ({comment = {}}) => {
 
-  const user = "Potter";
+  const {getUserData} = useAuth();
+  const currentUser = getUserData();
```

Comment:
Dumb question but have you tested this with the back end?

Reply:
Nope it will be test on the next pull request